server/databases.go

```diff
@@ -388,6 +427,81 @@ func (s *Service) DropRetentionPolicy(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusNoContent)
 }
 
+// Measurements lists measurements within a database
+func (h *Service) Measurements(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	srcID, err := paramID("id", r)
+	if err != nil {
+		Error(w, http.StatusUnprocessableEntity, err.Error(), h.Logger)
+		return
+	}
+
+	limit, offset, err := validMeasurementQuery(r.URL.Query())
+	if err != nil {
+		Error(w, http.StatusUnprocessableEntity, err.Error(), h.Logger)
+		return
+	}
+
+	src, err := h.Store.Sources(ctx).Get(ctx, srcID)
+	if err != nil {
+		notFound(w, srcID, h.Logger)
+		return
+	}
+
+	db := h.Databases
```

Comment:
I don't think "Databases" is intuitive here -- do you connect to a particular source in the list of sources? Are these databases within a source? Not clear to me without additional context.

Reply:
i agree, but changing that would imply the need to make its corollaries renamed as well, and this would even better be done as a refactor of the structure of our services. all of this is outside of the scope of this PR, but what i did in the meantime is rename `db` there to `dbsvc` to be clear contextually that this is a service.